src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -408,7 +413,7 @@ private IEnumerable<FileInfo> FindTemplateFilesInDirectory(DirectoryInfo directo
                     MatchCasing = MatchCasing.CaseInsensitive,
                     RecurseSubdirectories = true
                 }
-            ).Where(IsValidTemplate);
+            ).Where(s => !s.Name.Contains(".parameters")).Where(IsValidTemplate);
```

Comment:
Was this change necessary or added just in case? We should have a test that validates AnalyzeDirectory works properly when there are parameters files in the folder

Reply:
While testing I noticed that the `FindTemplateFilesInDirectory` method would also pick up the parameters files which could then cause issues if these are invalid. Seemed logical to me to try and keep the `FindTemplateFilesInDirectory` method to only try and find the template files, not the parameters, but easily reverted in you think its not necessary 😉

Reply:
Oh D: Yes, makes sense, we can add a test for that later, we should have done it before